installPhase.sh

```diff
@@ -15,11 +15,19 @@ else
     VERSION_APPENDIX="-NOGIT"
 fi
 
+# Extract version from Cargo.toml using toml2json
+PACKAGE_VERSION=$(toml2json < Cargo.toml | jq -r .package.version)
+if [ -z "$PACKAGE_VERSION" ]; then
+    echo "Could not extract version from Cargo.toml"
+    exit 1
+fi
+PACKAGE_VERSION+=$VERSION_APPENDIX
+
 mkdir -p $out
 cp README.md $out/
 cp -r ./pkg/* $out/
 cat package.json \
-| jq '{ name, version, collaborators, license } 
+| jq --arg ver "$PACKAGE_VERSION" '.version = $ver 
 | .repository = {"type": "git","url": "https://github.com/noir-lang/acvm-simulator-wasm.git"} 
 | .sideEffects = false | .files = ["nodejs","web","package.json"] 
 | .main = "./nodejs/acvm_simulator.js" 
```

Comment:
I didn't really see much sense in injecting these fields here so I've added most of them to [package.json](https://github.com/noir-lang/acvm-simulator/blob/master/package.json).

The only differences are that we
-  include the url of this repository rather than `acvm-simulator-wasm` as that's more useful.
- no longer add a git commit suffix to the version as I foresee that we'll be able to do stable releases of this package relatively soon.
  - `acvm-simulator-wasm` can add a suffix but it should be injected in that repository rather than here.